src/rust/ide/src/model/suggestion_database.rs

```diff
@@ -98,6 +101,21 @@ impl Entry {
     pub fn with_name(self, name:impl Into<String>) -> Self {
         Self {name:name.into(),..self}
     }
+
+    /// Generates HTML documentation for documented suggestion
+    fn gen_doc(doc: Option<String>) -> Option<String> {
```

Comment:
The None actually indicates an error. So this method should return FallibleResult<String>

Then, during conversion we should also log an error when we are unable to parse documentation (and leave `documentation` field as None - as it is now).